src/Index.php

```diff
@@ -272,10 +275,7 @@ public function getDocument($id, array $options = []): Document
             $data = [];
         }
 
-        $document = new Document($id, $data, $this->getName());
-        $document->setVersion($result['_version']);
-
-        return $document;
+        return new Document($id, $data, $this->getName());
```

Comment:
We can probably add the `seq_no`  and `primary_term` to document too as it's documented as returned: https://www.elastic.co/guide/en/elasticsearch/reference/7.9/docs-get.html#docs-get-api-response-body

Reply:
I was debating doing that since the param for the document update is different, I think it needs to be separated between what we get back and what is sent for a document update